locust-llm-benchmark/benchmark.py

```diff
@@ -0,0 +1,84 @@
+import json
+import os
+import time
+
+from locust import HttpUser, between, events, task
+
+PROMPT = "Explain MLOps in 300 tokens or less."
+TFY_HOST = os.getenv("TFY_HOST", "https://llm-gateway.truefoundry.com")
```

Comment:
TFY_HOST and TFY_API_KEY are usually used when we want to authenticate our client library. 

this code is normal locust.